### PR TITLE
Add 4.14 upgrade from dev-preview 4.14 on amd64/arm64/multiarch

### PR DESCRIPTION
**Pull request by shellyyang1989**
@jianlinliu @jiajliu @JianLi-RH  @jhou1 for review. Thank you! cc @jianping-shu

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-devpreview-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+- as: gcp-ipi-disconnected-private-workload-identity-f28
+  cron: 45 7 29 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.13 and @gcp-ipi and not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.13 and @gcp-ipi and not @fips'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-devpreview-4.14.yaml:444]: openshift-upgrade-qe-test-disconnected?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-devpreview-4.14.yaml:444]: Good catch, seem like this issue also exists in openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml.

@shellyyang1989 can you help fix both?

Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-devpreview-4.14.yaml]: Good catch! Updated on both 4.13 and 4.14

**Pull request merged**